index.bs

```diff
@@ -478,7 +478,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 1. Let |options| be the value of |options|["{{CredentialRequestOptions/scoped}}"].
 
 1. If any of the {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/rp}}, the
-    {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/user}}, or the {{ScopedCredentialEntity/id}}
+    {{ScopedCredentialEntity/name}} member of |options|.{{MakeCredentialOptions/user}},
+    the {{ScopedCredentialUserEntity/displayName}} member of |options|.{{MakeCredentialOptions/user}},
+    or the {{ScopedCredentialEntity/id}}
```

Comment:
do we really want to require RPs to pass all this in regardless of the authenticator type ?  This is needed for a multi-factor (aka 1st-factor) authnr, but not necessarily for single-factor (aka 2nd factor) authnr, yes?